src/main/scala/de/frosner/ddq/constraints/ConstraintUtil.scala

```diff
@@ -0,0 +1,13 @@
+package de.frosner.ddq.constraints
+
+import scala.util.Try
+
+object ConstraintUtil {
+
+  def tryToStatus[T](tryObject: Try[T], successCondition: T => Boolean): ConstraintStatus =
+    tryObject.map(
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/frank_7/drunken-data-quality/file/2305635415/issues/source?bid=3033737&fileBranchId=3196266#l8)